browser-test/src/support/ti_dashboard.ts

```diff
@@ -41,25 +41,25 @@ export class TIDashboard {
 
     await this.page.getByRole('button', {name: 'Save'}).click()
     await waitForPageJsLoad(this.page)
-    await this.expectSuccessAlertOnAddNewClient()
     await this.gotoTIDashboardPage(this.page)
     await waitForPageJsLoad(this.page)
   }
 
   async createMultipleClients(nameBase: string, copies: number) {
+    await this.page.getByRole('link', {name: 'Add new client'}).click()
```

Comment:
We don't need to return to the TI dashboard between adding each client.  We can stay on the "Add client" page.